public/robots.txt

```diff
@@ -0,0 +1,5 @@
+# See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
+#
+# To ban all spiders from the entire site uncomment the next two lines:
+# User-agent: *
+# Disallow: /
```

Comment:
need to determine the actual policy. i believe that the transition pages are not to be indexed but the landing pages yes